data/saas/config/adyen_config.yml

```diff
@@ -0,0 +1,74 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Adyen
+  type: adyen
+  description: A sample schema representing the Adyen integrations for Fides
+  user_guide: https://docs.ethyca.com/user-guides/integrations/saas-integrations/adyen
+  version: 0.1.0
+
+  connector_params:
+    - name: domain_management
+      label: Management domain
+      description: Please note that Adyen has both TEST and LIVE environments. E.g. management-test.adyen.com or management-live.adyen.com, and that API keys for live and test will not be the same.
+    - name: domain_ca
+      label: CA domain
+      description: Please note that Adyen has both TEST and LIVE environments. E.g. ca-test.adyen.com or ca-live.adyen.com, and that API keys for live and test will not be the same.
+    - name: merchant_account
+      label: Merchant account
```

Comment:
We needed the `merchant_account` for the delete body but we didn't have it in the `connector_param`, meaning we weren't asking the user for it 😊. Make sure you add the proper description.

Reply:
I had left that for later, shouldn't have!